src/openzaak/config/bootstrap/authorizations.py

```diff
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2024 Dimpact
+import sys
+from contextlib import contextmanager
+from io import StringIO
+
+from django.conf import settings
+from django.core.management import call_command
+
+import yaml
+from django_setup_configuration.configuration import BaseConfigurationStep
+from vng_api_common.models import JWTSecret
+
+from openzaak.components.autorisaties.models import Applicatie
+
+
+@contextmanager
+def override_stdin(data):
+    original_stdin = sys.stdin
+    sys.stdin = data
+    try:
+        yield
+    finally:
+        sys.stdin = original_stdin
+
+
+class AuthorizationConfigurationStep(BaseConfigurationStep):
+    """
+    Configure the Applicaties and Autorisaties.
+
+    From: https://open-zaak.readthedocs.io/en/stable/manual/api-authorizations.html
+    """
+
+    verbose_name = "Authorization configuration"
+    required_settings = [
+        "AUTHORIZATIONS_CONFIG_FIXTURE_PATH",
+    ]
+    enable_setting = "AUTHORIZATIONS_CONFIG_ENABLE"
+
+    def is_configured(self) -> bool:
+        with open(settings.AUTHORIZATIONS_CONFIG_FIXTURE_PATH) as f:
+            data = yaml.safe_load(f.read())
+            existing_uuids = set(
+                str(uuid) for uuid in Applicatie.objects.values_list("uuid", flat=True)
+            )
+            applicatie_uuids = {
+                row["fields"]["uuid"]
+                for row in data
+                if row["model"] == "authorizations.applicatie"
+            }
+            applicaties_configured = applicatie_uuids.issubset(existing_uuids)
+
+            existing_identifiers = set(
+                str(uuid)
+                for uuid in JWTSecret.objects.values_list("identifier", flat=True)
+            )
+
+            identifiers = {
+                row["fields"]["identifier"]
+                for row in data
+                if row["model"] == "vng_api_common.jwtsecret"
+            }
+            secrets_configured = identifiers.issubset(existing_identifiers)
+
+        return applicaties_configured and secrets_configured
+
+    def configure(self):
+        with open(settings.AUTHORIZATIONS_CONFIG_FIXTURE_PATH) as original_file:
+            content = original_file.read()
+
+            if settings.AUTHORIZATIONS_CONFIG_DOMAIN_MAPPING_PATH:
```

Comment:
Have you discussed the shape of mapping file with Joeri or Sjoerd? It looks like a bit of overkill, it could be just an env var `AUTHORIZATIONS_CONFIG_TARGET_DOMAIN` or something like this
On the other hand we make these commands for devops people. so I don't know what it more convenient for them

Reply:
I have discussed it with Joeri, it could indeed be a single target domain, but the thing is that we do support the usage of external catalogi as well, so if we have an import file that contains multiple different domains, the import wouldn't hold up.

I do still wonder if this is actually used anywhere in practice though, maybe we should give two options to simplify the config for instances that don't use external catalogi:
- either supply a `AUTHORIZATIONS_CONFIG_TARGET_DOMAIN` and just map all URLs to that domain (this assumes that the import file does not contain any external catalogi
- or supply a mapping file in case there are external catalogi?

Reply:
Since the mapping file construction is already there and documented, I'll leave it in